src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/ad/AutoplayVuap.java

```diff
@@ -126,9 +142,11 @@ private void clickByJS(By selector) {
   }
 
   private void clickElementInsideAd(By selector) {
-    // It needs to be clicked by JS, because our templates elements covers each other
-    // and there is no way to click it by just .click()
-    usingAdFrame(() -> clickByJS(selector));
+    usingAdFrame(() -> {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove useless curly braces around statement (sonar.java.source not set. Assuming 8 or greater.) [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1602)